lib/iex/lib/iex/autocomplete.ex

```diff
@@ -125,6 +128,46 @@ defmodule IEx.Autocomplete do
     end
   end
 
+  defp get_signatures!(name, aliases) when is_list(aliases) do
+    {:docs_v1, _, _, _, _, _, docs} =
+      aliases
+      |> Module.safe_concat()
+      |> Code.fetch_docs()
+
+    docs
+    |> Enum.filter(&match?({{:function, ^name, _}, _, _, _, _}, &1))
+    |> Enum.map(fn {_, _, [signature], _, _} -> signature end)
```

Comment:
According to EEP, this can be multiple entries which you join by line or whitespace. I believe you should write this:

```suggestion
    |> Enum.map(fn {_, _, signature, _, _} -> Enum.join(signature, " ") end)
```

Reply:
Is it possible to specify how you want to join them? (like new line vs whitespace)